packages/larva-tokens/build/deadline.json

```diff
@@ -0,0 +1,41 @@
+{
```

Comment:
This is also an extra file, please delete – the only deadline.json should be in src/brands/